src/utils/discordUtil.ts

```diff
@@ -4,6 +4,9 @@ import {getSettings} from '../database/models/SettingsModel';
 import {DEFAULT_PREFIX} from '../config';
 import {container, SapphireClient} from '@sapphire/framework';
 
+const ms = require('ms');
+const fourteenDaysInMs = 14*24*3600*1000;
```

Comment:
I would rather you use moment().subtract(14, 'd') here and grab the value from that object, rather than subtracting this number from the current timestamp.